.github/workflows/pr-validation.yml

```
@@ -32,7 +32,7 @@ jobs:
             ~/.cache/coursier/v1
           key: ${{ runner.os }}-sbt-ivy-cache-${{ hashFiles('**/build.sbt') }}-${{ hashFiles('**/plugins.sbt') }}-${{ hashFiles('**/build.properties') }}
       - name: Compile and test
-        run: sbt -batch "; docs/mdoc; +test; +doc; version"
+        run: sbt -batch "; docs/mdoc; +test; +doc; version; mimaReportBinaryIssues"
```

Batch is useless - also let's check all Scala versions
```suggestion
        run: sbt docs/mdoc +test +doc version +mimaReportBinaryIssues
```